index.html

```diff
@@ -0,0 +1,186 @@
+            </path>
+          </g>
+        </svg>
+      </div>
+      <div class="d-flex justify-content-between align-items-center ">
+        <span class="d-lg-none"></span>
+        <span class="d-lg-none"></span>
+        <span class="d-lg-none"></span>
+        <a href="#" class="d-none d-lg-inline-block">LOGIN</a>
+        <button class="btn btn-warning roboto-font rounded-pill">
+          SUBSCRIBE &rsaquo;
+        </button>
+        <div class="hambutt d-lg-none">
+          <div></div>
+          <div class="ham-2"></div>
+          <div class="ham-3"></div>
+        </div>
+      </div>
+    </div>
+    <nav class="border-bottom d-none d-lg-block">
+      <div class="container-fluid">
+        <ul class="nav p-1 fw-bolder">
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">U.S<span class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">World<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Businnes<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Tech & Science<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Culture<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">NewsGeek<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Sports<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Health<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">The Debate<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Experts<span
+                class="border-end mx-3"></span></a>
+          </li>
+          <li class="navbar-brand me-0">
+            <a class="roboto-font text-decoration-none text-dark" href="#">Vantage</a>
+          </li>
+          <li class="fs-6 mt-2 mx-2 text-black-50 fw-light">Search</li>
```

Comment:
- [ ] You are missing the search icon here, can you please add a magnifying glass just so it is like the original? 
- [ ] This should also be wrapped in a `form` with an `input` tag, so I could type anything I need to search on your webpage.